### DISCUSSION TITLE
Fil rolünü oynayanlar sayısı

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#234] [Zeka Sorusu]
Merve, Özlem, Ekrem, Gökhan ve Murat bir oyun oynuyorlar. Her oyuncu ya bir fil ya da bir karınca rolündedir. Filler asla gerçeği söylemezler, karıncalar ise daima gerçeği söyler:
Merve: "Özlem karıncadır."
Ekrem:"Gökhan bir fildir."
Murat:"Merve fil değildir."
Özlem:"Ekrem karınca değildir."
Gökhan:"Merve ve Murat farklı rollerdedir."
Yukarıdaki metinlerde oyuncuların konuşmalarına kaç tane fil rolünü oynayan vardır?

**Comment by ElifNurSoyler** [comment]
Merve, Murat, Özlem ve Gökhan = FİL
Ekrem = KARINCA
Cevap = 4
